pkg/component/component.go

```diff
@@ -60,21 +60,41 @@ func CreateFromDir(client *occlient.Client, name string, ctype string, dir strin
 
 // Delete whole component
 func Delete(client *occlient.Client, name string, applicationName string, projectName string) (string, error) {
+
 	cfg, err := config.New()
 	if err != nil {
-		return "", errors.Wrapf(err, "unable to delete component %s", name)
+		return "", errors.Wrapf(err, "unable to create new configuration to delete %s", name)
 	}
 
 	labels := componentlabels.GetLabels(name, applicationName, false)
 
 	output, err := client.Delete("all", "", labels)
 	if err != nil {
-		return "", errors.Wrapf(err, "unable to delete component %s", name)
+		return "", errors.Wrapf(err, "error deleting component %s", name)
 	}
 
-	err = cfg.SetActiveComponent("", projectName, applicationName)
+	// If there are components that still exist (len > 0) then set it to the default component..
+	components, err := List(client, applicationName, projectName)
 	if err != nil {
-		return "", errors.Wrapf(err, "unable to delete component %s", name)
+		return "", errors.Wrapf(err, "unable to retrieve list of components")
+	}
+
+	// If there's more than one component, set it to the first one..
+	if len(components) > 0 {
+		err = cfg.SetActiveComponent(components[0].Name, applicationName, projectName)
+		log.Warnf("No active component found, setting active component to %s", components[0].Name)
```

Comment:
This shouldn't be a warning. This is normal operation.
Normal message saying that `Setting active component to %s", components[0].Name.` Is enought.